pyspark/bigdl/keras/issues.md

```diff
@@ -0,0 +1,33 @@
+## Summary
+Up to now, we have generally supported __ALL__ the layers of [__Keras 1.2.2__](https://faroit.github.io/keras-docs/1.2.2/) to be loaded into BigDL.
+
+Self-defined Keras layers or [`Lambda`](https://faroit.github.io/keras-docs/1.2.2/layers/core/#lambda) layers are not supported for now. Weight sharing is not supported for now.
+
+This pages lists the so-far unsupported arguments for specific layers and some known issues.
+
+## Unsupported Layer Arguments
+For the following arguments, only the default values are expected and supported.
```

Comment:
currently only the default values are expected and supported

Reply:
Updated.